src/test/java/uk/gov/hmcts/reform/hmc/validator/CapitalizedEnumPatternValidatorTest.java

```diff
@@ -0,0 +1,59 @@
+package uk.gov.hmcts.reform.hmc.validator;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.reform.hmc.model.DayOfWeekUnAvailableType;
+import uk.gov.hmcts.reform.hmc.model.UnavailabilityDow;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import javax.validation.ConstraintViolation;
+import javax.validation.Validation;
+import javax.validation.Validator;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@ExtendWith(MockitoExtension.class)
+class CapitalizedEnumPatternValidatorTest {
+
+    static Validator validator;
+
+    @BeforeEach
+    void setUp() {
+        validator = Validation.buildDefaultValidatorFactory().getValidator();
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = {"MONDAY", "TUESDAY", "WEDNESDAY", "THURSDAY", "FRIDAY", "SATURDAY", "SUNDAY", "MoNdAy",
+            "TUEsday", "wednesday", "THuRSDAY", "FridaY", "saturday", "sUNDAY"})
+    void whenInvalidCaseUnavailabilityDow(String dowValue) {
+        final Set<ConstraintViolation<UnavailabilityDow>> violations =
+                createAndValidateUnavailabilityDow(dowValue);
+        assertFalse(violations.isEmpty());
```

Comment:
Lines 38 - 42
I can see this pattern has been used in some other tests - wondering if this could be extracted into a function that will be like:
`violations->mapToListOfMessages->assertThatTheListSatisfiesMyExpectations`
Just to make it a bit easier to read

Reply:
I will leave it as is for now given there are only 2 tests, but can refactor if it becomes unmanageable or completely unreadable